### PR TITLE
Update ar-EG.txt (Don't Merge It yet!)

### PR DESCRIPTION
**Pull request by Omaranwa**
Applying for issue(s):
- #2386
- #2387
- #2388
- #2430
- #2466
- #2472
- #2492
- #2501
- #2509

EDIT (by tupaschoal) - These had already been done previously and are not covered in this PR:
- #2345
- #2354
- #2363
- #2369
- #2378
- #2399
- #2414

### CODE REVIEW

File: data/language/ar-EG.txt [hunk 1]
```diff
@@ -179,6 +180,7 @@ STR_0602    :Roller coaster trains are accelerated out of the station by linear
 STR_0603    :أفعوانية خشبية مع مسار فولاذي، تسمح بإنحدارت وإنعكاسات حادة.
 STR_0604    :يركب الراكبون سيارة صغيرة على مسار أحادي ضيق، حيث يتسابقون عبر طرق مُنعكسة وملفات حادة.
 STR_0605    :يتزلق الراكبون على مسار فولاذي متعرج، مع إستخدام المكابح للتحكم في السرعة
+STR_0606    :أفعوانية خشبية قديمة الطراز تتميز بركوب سريع ووعر، مع الكثير من التعلق بالهواء، وقوة تسارع جانبية قوية مصممة لتشعر الراكب بإنه "خارج عن السيطرة"
```
Comment (tupaschoal) [inline data/language/ar-EG.txt:183]: The end of this seems to use normal plain quotes "", whereas the original uses curly single quotes:
> ‘out-of-control’

Is this change intended to be language-specific or accidental?

**Review by tupaschoal — COMMENTED** [review]
Some issues had already been done: 2345, 2354, 2363, 2369, 2378, 2399, 2414 . Any reason for including them in the PR description? (I have edited the description for now to distinguish)

Aside from that everything that is in the diff seems to be ok, can you elaborate on why to wait for merge?

File: data/language/ar-EG.txt (hunk 1 above)
Comment (Omaranwa) [reply data/language/ar-EG.txt:183]: Yea, In the Arabic language, double quotes are used.

**Comment by Omaranwa** [comment]
> Some issues had already been done: 2345, 2354, 2363, 2369, 2378, 2399, 2414 . Any reason for including them in the PR description? (I have edited the description for now to distinguish)
> 
> Aside from that everything that is in the diff seems to be ok, can you elaborate on why to wait for merge?
the thing is that these issues are done, but not marked!
I had to add them for reference.

**Comment by tupaschoal** [comment]
> * for the WIP, I plan to add more in-line translations in the coming days!

Ok! I'll mark the PR as draft in the meantime, but feel free to mark it as ready for review when you finish! Keep in mind that it is better for us to do small batches than to do one big chunk :)

**Comment by Omaranwa** [comment]
> > * for the WIP, I plan to add more in-line translations in the coming days!
> 
> Ok! I'll mark the PR as draft in the meantime, but feel free to mark it as ready for review when you finish! Keep in mind that it is better for us to do small batches than to do one big chunk :)



> > * for the WIP, I plan to add more in-line translations in the coming days!
> 
> Ok! I'll mark the PR as draft in the meantime, but feel free to mark it as ready for review when you finish! Keep in mind that it is better for us to do small batches than to do one big chunk :)

Ok, Thanks for that fast reply, this pull is a rewrite of some sentences here and there, plus some more strings in line with other strings (a lot of words have a double meaning, which makes the translation weird).

**Review by tupaschoal — COMMENTED** [review]
Seems ok up until the last commit (6ec76ef67401b57b7172131a50a5d155fcdc5e47)

**Comment by tupaschoal** [comment]
@Omaranwa can we merge this one as is, and then you resume your work on another PR? It's already quite big, it will be easier to review in smaller chunks.

**Review by Gymnasiast — APPROVED** [review]
